github_app_installation.go

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package tfe
+
+import (
+	"context"
+	"fmt"
+	"net/url"
+)
+
+// Compile-time proof of interface implementation.
+var _ GHAInstallations = (*gHAInstallations)(nil)
+
+// GHAInstallations describes all the Github App Installations related methods that the
+// Terraform Enterprise API supports.
+//
+// TFE API docs:
+// https://www.terraform.io/docs/cloud/api/gha-installations.html
```

Comment:
This documentation link didn't work. Is this API documented yet? There's no rules around which should exist first, but I'm trying to figure out if this feature is stable, generally available, etc